xblockutils/base_test.py

```diff
@@ -0,0 +1,89 @@
+import os
+import sys
+import time
+
+import pkg_resources
+
+from django.template import Context, Template
+
+from selenium.webdriver.support.ui import WebDriverWait
+
+from workbench import scenarios
+from workbench.test.selenium_test import SeleniumTest
+
+from .resources import ResourceLoader
+
+
+class SeleniumBaseTest(SeleniumTest):
```

Comment:
Sounds great @bradenmacdonald, does the rename mean that we'll need to replace existing classes that inherit from ```SeleniumBaseTest```, or does that remain and we can port over at leisure?